homeassistant/components/sensor/daikin.py

```diff
@@ -0,0 +1,98 @@
+"""
+Support for Daikin AC Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.daikin/
+"""
+import logging
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (
+    CONF_HOST, CONF_NAME, CONF_ICON,
+    CONF_MONITORED_CONDITIONS, CONF_TEMPERATURE_UNIT
+)
+import homeassistant.helpers.config_validation as cv
+
+from homeassistant.components.daikin import (
+    SENSOR_TYPES,
+    manual_device_setup
+)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_NAME, default=None): cv.string,
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=SENSOR_TYPES.keys()):
+        vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
+})
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Daikin sensors."""
+    if discovery_info is not None:
+        host = discovery_info.get('ip')
+        name = None
+        monitored_conditions = discovery_info.get(
+            CONF_MONITORED_CONDITIONS, list(SENSOR_TYPES.keys())
+        )
+        _LOGGER.info("Discovered a Daikin AC sensor on %s", host)
+    else:
+        host = config.get(CONF_HOST)
+        name = config.get(CONF_NAME)
+        monitored_conditions = config.get(CONF_MONITORED_CONDITIONS)
+        _LOGGER.info("Added Daikin AC sensor on %s", host)
+
+    device = manual_device_setup(hass, host, name)
+
+    sensors = []
+    for monitored_state in monitored_conditions:
+        sensors.append(DaikinClimateSensor(device, name, monitored_state))
+
+    add_devices(sensors, True)
+
+
+class DaikinClimateSensor(Entity):
```

Comment:
What will this class represent? Are these pyhsical sensors, i.e., small devices attached to a wall? Or is this just reading data from the AC device (which is represented by the climate platform)? In the latter case I think these values should rather be attributes of the climate device, not its own sensor.

Reply:
They are not real devices but attributes of the real device, so it's the 2nd case.
If the climate device itself would have (and already has some of the attributes, eg: current_temperature), how can I expose this as a dedicated sensor because I don't know other way then creating this sensor.

PS: Of course if I use automatation script I could use the climate device attributes, but it's not that case.

Reply:
Just use a template sensor, and set the template to
 `{{ states.climate.your_entity_id.attributes.current_temperature }}`

Reply:
@tinload u're basically saying to drop the dedicated daikin climate sensor, right ?
If yes, do I need to add to add any code in the climate component?

Reply:
@tinloaf Some Daikin models report both indoor and **outdoor** temperature (and in some cases even indoor humidity), I think it would be neat if those sensors would show up as a sensor "badge" that can be used in different groups.

Reply:
@tinloaf any updates on this topic to know how we should proceed.
Ineed @fredrike is right and if we look also on https://github.com/home-assistant/home-assistant/pull/11095 indeed we need those dedicated sensors to be able to share them with Google Assistant

Reply:
Well, opinions on this topic (not specific to this platform) are currently forming. :laughing: I'd say go ahead and leave it as a separate sensor.

Reply:
Perfect :)